telemetry/definitions/commonDefinitions.json

```diff
@@ -878,6 +883,18 @@
             ],
             "passive": true
         },
+        {
+            "name": "aws_refreshCredentials",
+            "description": "Emitted when credentials are automatically refreshed by the AWS SDK or Toolkit",
+            "passive": true,
+            "metadata": [
+                { "type": "result" },
+                { "type": "reason", "required": false },
+                { "type": "credentialType", "required": false },
+                { "type": "credentialSourceId", "required": false },
```

Comment:
This should be `awsId` for builder ID and `iamIdentityCenter` for "enterprise SSO"